docs/application/dotnet/getting-started/runtime.md

```diff
@@ -0,0 +1,24 @@
+<a id="runtime"></a>
+# Managed runtime Advantages
```

Comment:
Change: Managed runtime Advantages 
to
Managed Runtime Advantages 

If you are following title case, then please follow the style throughout.

Reply:
I fix it.